docs/contributing.rst

```diff
@@ -80,6 +80,30 @@ commits and push them to GitHub.
 If you've pushed your changes to GitHub already you'll need to force push your branch
 after this with ``git push -f``.
 
+Commit Message formatting
+-------------------------
+To ensure that all commit messages in the master branch follow a specific format, we
+enforce that all commit messages must follow the following format:
+
+.. code-block:: bash
+
+  FEAT-GH9999: Add `DataFrame.rolling` functionality, to enable rolling window operations
```

Comment:
Isn't the `GH` prefix redundant?

Reply:
Yes, the purpose is link issues to the commits and the PRs, but it seems GitHub cannot do this with commit messages and the GH prefix. I will change it so GitHub can link them with the commits, it is important.